ch_backup/logic/access.py

```diff
@@ -28,13 +33,39 @@ def backup(self, context: BackupContext) -> None:
         """
         Backup access control entities.
         """
-        objects = context.ch_ctl.get_access_control_objects()
-        context.backup_meta.set_access_control(objects)
-        access_control = context.backup_meta.access_control
 
-        if self._has_replicated_access(context):
-            self._backup_replicated(access_control.acl_ids, context)
-        self._backup_local(access_control.acl_ids, context)
+        clickhouse_access_path = context.ch_ctl_conf["access_control_path"]
+        backup_tmp_path = os.path.join(
+            context.ch_ctl_conf["tmp_path"], context.backup_meta.name
+        )
+        user = context.ch_ctl_conf["user"]
+        group = context.ch_ctl_conf["group"]
+
+        create_dir(clickhouse_access_path, user, group)
```

Comment:
1. Actually, I don't see a necessity of creating separate function for this (creating folder and chown it). Two separate commands are relatively expressive.
2. `clickhouse_access_path` does not exist here ?

Reply:
> clickhouse_access_path does not exist here ?

Yes, clickhouse lazily creates `/var/lib/clickhouse/access`

Reply:
>clickhouse_access_path does not exist here ?

Yes, clickhouse lazily creates `/var/lib/clickhouse/access`